FrontEnd/src/UserAuthentication/Auth.js

```diff
@@ -44,7 +44,7 @@ const loginUser = async (email, password) => {
     }
     return user;
   } catch (error) {
-    console.error("Error logging in:", error);
+    throw new Error("Error logging in");
```

Comment:
What will user see if an error is thrown? Will the app crash or frozen? Can users try again?

Reply:
User's get a specific error message and they can try again